bmc_remedy_itsm/help.md

```diff
@@ -395,7 +529,7 @@ Example output:
       "Local Phone": "555-5454",
       "Extension": "66",
       "Street": "1114 Eighth Avenue, 31st Floor",
-      "Internet E-mail": "blah@example.com",
+      "Internet E-mail": "J.Unser@calbroservices.com",
```

Comment:
We're standardizing on `user@example.com` for all e-mail addresses in plugin examples.

Reply:
Update for instances in this file.